Version.h

```diff
@@ -1,6 +1,12 @@
 #pragma once
 
+#include "StdInc.h"
```

Comment:
Why do we need it here? It is precompiled header.

Reply:
To gain access to the namespace macro, otherwise there're build errors.